docker/docker-compose.1604.53.yaml

```diff
@@ -36,7 +36,7 @@ services:
       - MAX_ALLOCS_ALLOWED_udp_1000_reqs_1_conn=16250
       - MAX_ALLOCS_ALLOWED_udp_1_reqs_1000_conn=226050
       - MAX_ALLOCS_ALLOWED_1000_tcpbootstraps=4100
-      - MAX_ALLOCS_ALLOWED_1000_tcpconnections=208100
+      - MAX_ALLOCS_ALLOWED_1000_tcpconnections=223100
```

Comment:
that's interesting. Why is this +15 allocs and the test case that nominally does _more (`1_reqs_1000_conn`) only gains + 1? Any ideas @PeterAdams-A ?

Reply:
ping @Lukasa / @PeterAdams-A

Reply:
It's now +2 vs +1.